exporter/awsxrayexporter/translator/cause_test.go

```diff
@@ -629,3 +629,66 @@ TypeError: must be str, not int`
 	assert.Equal(t, "main.py", *exceptions[0].Stack[1].Path)
 	assert.Equal(t, 14, *exceptions[0].Stack[1].Line)
 }
+
+func TestParseExceptionWithJavaScriptStacktrace(t *testing.T) {
```

Comment:
I think we are missing some types of frames in the tests (will review the parsing logic after we have all the tests represented :-).

https://nodejs.org/api/errors.html#errors_error_stack

Don't see one that is missing the method name (`at /home/gbusey/file.js:525:2` in the example). And it says location can be the word `native` so we need examples like `forEach (native)` and just `native` I think.

Reply:
Noted - making changes